FluidNC/src/Protocol.cpp

```diff
@@ -122,8 +123,11 @@ char activeLine[Channel::maxLine];
 
 bool pollingPaused = false;
 void polling_loop(void* unused) {
+    // Add current task to the watchdog
```

Comment:
Ouch, just realized that this creates an ESP32 dependency in code that should be CPU agnostic.  Hmmm.